lib/fs/basicfs_windows.go

```diff
@@ -150,3 +151,31 @@ func (f *BasicFilesystem) Roots() ([]string, error) {
 
 	return drives, nil
 }
+
+func (f *BasicFilesystem) resolveWin83(absPath string) string {
+	if !strings.Contains(absPath, "~") {
+		return absPath
+	}
+	if in, err := syscall.UTF16FromString(absPath); err != nil {
+		out := make([]uint16, 2*len(absPath))
+		n := uint32(len(out))
+		for {
+			n, err = syscall.GetLongPathName(&in[0], &out[0], n)
```

Comment:
Nit: `n` has two meanings: it starts off being the size of the buffer, and then becomes the return value from `GetLongPathName`.